src/CircleEditor/MessageType.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Custom message types to communicate with vscode and webviews
+ */
+
+export interface ResponseModel {
+  command: string;
+  type: string;
+  offset: number;
+  length: number;
+  total: number;
+  responseArray: Uint8Array;
+}
+
+export interface RequestMessage {
+  command: string;
+  type: string;
+  data: any;
+}
+
+export interface CustomInfoMessage {
+  command: string;
+  data: any;
+}
```

Comment:
This is exactly same with `ResponseJson`.
Could you let me know about the difference? :)

Reply:
ReponseJson is used when model of json format is sent to webview. CustomInforMessage is used to send various information about custom operators. I think it is better to merge them into one interface. I will change this part.